src/test/java/edu/kis/vh/nursery/RhymersJUnitTest.java

```diff
@@ -29,27 +30,29 @@ public void testCallCheck() {
 
 	@Test
 	public void testIsFull() {
-		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+		final DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
 		final int STACK_CAPACITY = 12;
 		for (int i = 0; i < STACK_CAPACITY; i++) {
-			boolean result = rhymer.isFull();
+			final boolean result = rhymer.isFull();
 			Assert.assertEquals(false, result);
 			rhymer.countIn(888);
 		}
 
-		boolean result = rhymer.isFull();
+		final boolean result = rhymer.isFull();
 		Assert.assertEquals(true, result);
 	}
 
 	@Test
 	public void testPeekaboo() {
-		DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
-		final int EMPTY_STACK_VALUE = -1;
+		final DefaultCountingOutRhymer rhymer = new DefaultCountingOutRhymer();
+
+		//! CORRECTION -1 to 0 
+		final int EMPTY_STACK_VALUE = 0;
```

Comment:
The original constant should be used (it would be best to use a common constant in a stacks interface).